policyengine_us/parameters/gov/states/ri/tax/income/exemption/reduction/percentage.yaml

```
@@ -0,0 +1,41 @@
+description: Rhode Island phases this percentage of the total exemption amount based on modified agi.
```

Spell out `adjusted gross income` in descriptions, or capitalize `AGI` if in a label or comment, only `agi` if in a variable or parameter name
```suggestion
description: Rhode Island phases this percentage of the total exemption amount based on modified adjusted gross income.
```